pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -47,6 +47,8 @@ const (
 	kubernetesHostnameTopologyKey = "kubernetes.io/hostname"
 
 	portName = "--port"
+
```

Comment:
@ksimon1 I saw that there is some effort from the code-quality to suggest a style where we reduce the scope of the const to a single function when its used only there, please see a reference [here](https://github.com/kubevirt/kubevirt/pull/7983/commits/fcdeeb92a0e4fbf6fe5922138b615d3beac1029e)